finagle-redis/src/main/scala/com/twitter/finagle/redis/HyperLogLogCommands.scala

```diff
@@ -0,0 +1,47 @@
+package com.twitter.finagle.redis
+
+import _root_.java.lang.{Long => JLong,Boolean => JBoolean}
```

Comment:
I see. Sorry. Thanks for the explanation Travis! In this case we definitely should "follow the style" at least in this PR.